EIPS/eip-1.md

```diff
@@ -37,11 +37,31 @@ An EIP must meet certain minimum criteria. It must be a clear and complete descr
 
 ## EIP Work Flow
 
+### Shepherding an EIP
+
 Parties involved in the process are you, the champion or *EIP author*, the [*EIP editors*](#eip-editors), and the [*Ethereum Core Developers*](https://github.com/ethereum/pm).
 
-:warning: Before you begin, vet your idea, this will save you time. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be rejected based on prior research (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used. Examples of appropriate public forums to gauge interest around your EIP include [the Ethereum subreddit], [the Issues section of this repository], and [one of the Ethereum Gitter chat rooms]. In particular, [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your EIP.
+Before you begin writing a formal EIP, you should vet your idea. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. Proposing an EIP which has not been properly dilligenced may be grounds for EIP Editors not granting the "Draft" status. It is thus recommended to open a discussion thread on [the Ethereum Magicians forum] to do this, but you can also use [one of the Ethereum Gitter chat rooms], [the Ethereum subreddit] or [the Issues section of this repository]. 
```

Comment:
I'm not sure 'dilligenced' is a word.

Reply:
More substantively: I don't think we want to put the burden on EIP editors of determining if a submission is "good enough to be a draft". We've been trying to make the draft step purely based on being correctly formatted, and I think for everyone's sanity, that's the best option. More editorial control can be exercised on drafts that want to advance to Last Call or Final status.

Reply:
Thanks for the feedback, @Arachnid. @madeoftin, I worded this section in a stronger way based on your feedback from earlier. What do you think of Nick's comments?

Reply:
@MadeofTin friendly ping :-)

Reply:
@Arachnid @MadeofTin based on the recent conversations on gitter around pushing for EIPs to be merged as drafts sooner rather than later (and now having 1/2 the discussion about them on a PR and the other on the `discussion-to` link), I think we should remove this sentence from the change:
> Proposing an EIP which has not been properly dilligenced may be grounds for EIP Editors not granting the "Draft" status.

What do you think?

Reply:
SGTM.